scripts/qa_crowbarsetup.sh

```diff
@@ -2047,6 +2047,8 @@ function reboot_nodes_via_ipmi
                 sleep $((50 + RANDOM % 20))
 
                 ipmitool -H $ip -U root -P $pw chassis bootdev pxe
+                # Please do not touch this sleep. It is important for the ipmi to work on qa hardware
+                sleep $((50 + RANDOM % 20))
```

Comment:
@vmoravec 
> Please do not touch this sleep. It is important for the ipmi to work on qa hardware

Sorry, but this says nothing about the cause for the sleep. The goal of having a comment is to describe the real cause. Anybody touching this code later should be able to determine that the line can be dropped if the underlying cause vanished (eg. hardware was replaced or hardware fixed). So, what is the cause?